app/main/views/users.py

```diff
@@ -439,19 +439,15 @@ def get_buyers_stats():
     return jsonify(buyers=buyers)
 
 
-def get_duplicate_users(email_address=None):
-    duplicate = {"duplicate": None}
-    if email_address is not None:
-        email_address = email_address
-    else:
-        json_payload = get_json_from_request()
-        json_has_required_keys(json_payload, ["email_address"])
-        email_address = json_payload["email_address"]
-
+@main.route('/users/checkduplicates', methods=['POST'])
+def get_duplicate_users():
```

Comment:
The method is called by the new signup page and the route is used by our legacy code until the react page is up